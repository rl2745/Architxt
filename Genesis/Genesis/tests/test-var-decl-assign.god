void init() {}
void update(int f) {}

int main() {
    int x; x = 3;
    int y; y = 4;

    print(x);
    print(y);

    float z;
    z = 3.14;
    printfl(z);

    return 0;
}
