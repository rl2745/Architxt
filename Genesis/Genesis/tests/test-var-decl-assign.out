3
4
3.140000
